db/sovereign-db/src/state_db.rs

```diff
@@ -17,6 +20,7 @@ use crate::{
 #[derive(Clone)]
 pub struct StateDB {
     db: Arc<DB>,
+    version: Arc<Mutex<Version>>,
```

Comment:
Can we change this to `last_version_written` as well?

Reply:
But this is `last_version_written +1`, so maybe `next_version`?

Reply:
We can do `next_version`!